src/features/projects/components/maps/TimeTravel.tsx

```diff
@@ -445,6 +445,123 @@ export default function TimeTravel({}: Props): ReactElement {
           }
         });
       }
+
+          // ESRI
+    if (selectedSource1 === 'esri') {
```

Comment:
I do not see the source `esri` when looking at the Vercel deployment - how can I test it?

<img width="338" alt="Bildschirmfoto 2021-08-17 um 10 55 56" src="https://user-images.githubusercontent.com/1532418/129695801-455bcbf2-bd57-48fc-94f2-18e1580b6bf0.png">

Reply:
I think it only appears after https://github.com/Plant-for-the-Planet-org/planet-geo-api/pull/8 got deployed, right?

Reply:
Yes, test here until then - https://planet-test-sarvesh.herokuapp.com/